deployment/E_heroku.md

```diff
@@ -163,7 +149,7 @@ config :logger, level: :info
 config :hello_phoenix, HelloPhoenix.Repo,
   adapter: Ecto.Adapters.Postgres,
   url: System.get_env("DATABASE_URL"),
-  pool_size: 20,
+  pool_size: String.to_integer(System.get_env("POOL_SIZE") || "10"),
```

Comment:
@jeregrine I tested it with and without `config_vars_to_export` in the buildpack configs. It seems that `POOL_SIZE` is only needed at compile time if `|| "10"` is left out. I verified that the Phoenix app actually uses the value of `POOL_SIZE` and not `"10"` in absence of `config_vars_to_export`.